src/Components/ClustersListTable/ClustersListTable.spec.ct.js

```diff
@@ -160,11 +156,11 @@ describe('clusters list table', () => {
     });
 
     it('sorting using last seen', () => {
-      // TODO create a function used also in other tests
-      cy.get(ROOT)
-        .find('th[data-label="Last seen"]')
-        .should('have.class', 'pf-c-table__sort pf-m-selected');
-      // TODO check window.location as in  RecsListTable (if applicable)
+      const column = 'Last seen';
+      tableIsSortedBy(column);
+      expect(window.location.search).to.contain(
```

Comment:
isn't https://github.com/RedHatInsights/ocp-advisor-frontend/pull/262/files#diff-11510ebd9fcd477bd2c0ab72124a29e1b5a7a8ec9743a472cbc67852ba71b74aR173 supposed to also have this check?

Reply:
which check? The `window.location`? It is not implemented for the other component

Reply:
yep, I meant window.location check. and yes, it's not implemented there, sorry for a noise :D